src/components/other/TabularBook/Sheet/index.js

```diff
@@ -397,12 +397,13 @@ export default class Sheet extends React.PureComponent {
             <div className={_cs(className, styles.tabularSheet, 'tabular-sheet')}>
                 <div className={styles.optionsBar}>
                     <Cloak
-                        hide={this.shouldHideEditButton}
+                        hide={obj => this.shouldHideEditButton(obj, disabled, fieldList)}
```

Comment:
Inline functions are highly discouraged.
The behavior can be achieved without using inline function.